core/src/components/input/input.ionic.scss

```diff
@@ -21,6 +21,35 @@
   min-height: 48px;
 }
 
+// Input Clear Button
+// ----------------------------------------------------------------
+
+.input-clear-icon {
+  width: 16px;
```

Comment:
Please update FW-6100 to make sure to update the width when input size is `xlarge`. That way it doesn't slip through the cracks.

Reply:
It looks like the button is 16px across all input sizes? At least according to the Figma. Screenshot from xlarge size:
![image](https://github.com/ionic-team/ionic-framework/assets/90629384/a3381bcf-da32-4396-9501-da4871490225)

Reply:
Thank you! I re-read the UX reqs and realized that I misread it. :face_palm: